playbooks/gluster-cluster/cluster-vars-2x2.yml

```diff
@@ -0,0 +1,73 @@
+---
+# Other supported disktypes are JBOD and RAID10
```

Comment:
RAID5, RAID6 ?

Reply:
RAID6 is mentioned in the example in the next line. I had missed RAID5